packages/block-editor/src/components/block-list/use-block-props/use-selected-block-event-handlers.js

```diff
@@ -22,7 +22,8 @@ import { store as blockEditorStore } from '../../../store';
 export function useEventHandlers( { clientId, isSelected } ) {
 	const { getBlockRootClientId, getBlockIndex } =
 		useSelect( blockEditorStore );
-	const { insertDefaultBlock, removeBlock } = useDispatch( blockEditorStore );
+	const { insertAfterBlock, insertDefaultBlock, removeBlock } =
```

Comment:
`insertDefaultBlock` is only passed as the dependency to the hook, so it should be removed from here and replaced with `insertAfterBlock` in other place.

Reply:
Good catch. I just changed that in [this commit](https://github.com/WordPress/gutenberg/pull/59361/commits/87870a2a34c9f1651fae866f0b29666e6957fed4).